src/Components/NavBar/NavBar.tsx

```diff
@@ -28,18 +32,31 @@ import {
   SoloIcon,
   Spacer,
 } from "@artsy/palette"
+import { TrackingProp } from "react-tracking"
 
 interface NavBarProps {
-  user?: User
+  tracking?: TrackingProp
 }
 
-export const NavBar: React.FC<NavBarProps> = () => {
+export const NavbarContext = React.createContext<{
+  tracking?: TrackingProp
+  Schema?: typeof Schema
```

Comment:
This `typof Schema` isn't actually enforcing anything, but I imagine we could use `Trackables` here

Reply:
Actually, `typeof Schema` did work.

Reply:
I think that wrapping the components with `track()` and importing the schema is better as it’s consistent with other places and doesn’t require the definition of a new context nor an unclear need [to a casual reader] for memoization.

Reply:
I removed the `useMemo` for now (since we're not yet concerned with changing values, and relocating tracking-specific stuff to a `useTracking` hook in [this commit](https://github.com/artsy/reaction/pull/2385/commits/f7bcc57ebcbf875efa1775ec0887d7934488fbec). In a follow up PR I'll spike on moving this pattern to SystemContext, and out of NavBar's util's folder. This will allow us to cleanly use tracking in FC components without needing to HOC wrap and import and mock.